test/autotests/markoc/single-template-write-version-comment/test.js

```diff
@@ -1,15 +1,11 @@
-var expect = require('chai').expect;
-var markoVersion = require('../../../../package.json').version;
-
-function _appendMarkoVersionComment(str) {
-    return '// Compiled using markoc@' + markoVersion + ' - DO NOT EDIT\n' + str;
+function removeMarkoVersionComment(str) {
+    return str.replace(/^.*\n/, '');
 }
 
 exports.test = function(helpers) {
     helpers.spawnSync(['template.marko']);
 
     var compiledFile = helpers.readSync('template.marko.js').toString();
-    var expectedFile = _appendMarkoVersionComment(helpers.readSync('expected.js'));
     compiledFile = compiledFile.replace(/marko\/dist\//g, 'marko/src/');
-    expect(compiledFile).to.deep.equal(expectedFile);
+    helpers.compare(removeMarkoVersionComment(compiledFile), '.js');
```

Comment:
In any other case, I would say stripping the comment is fine, but isn't this test supposed to be asserting the existence of the comment?

Reply:
Perhaps. I will take a closer look.